docs/user/filesystem.md

```diff
@@ -1021,17 +1021,16 @@ libpil4dfs intercepting summary for ops on DFS:
 [op_sum ]  5003
 ```
 
+### Bypassing function interception in libpil4dfs
+libpil4dfs could boost IO performance considerably over using dfuse for IO intensive workloads. However, initializing daos environment in libpil4dfs is expensive. To alleviate the overhead of libpil4dfs, users could set env to disable function interception in specific applications. I.e., let dfuse handle all IO requests. Users can set env "D_IL_BYPASS_LIST" to bypass function interceptions in target processes and their child processes. This is recommended for short-lived processes (shorter than 100 milliseconds) that run a large number (more than 1,000) of times. Example to set env:
```

Comment:
I would reword this a bit. it starts of with 2 contradicting statements:
1) libpil4dfs boost io performance over dfuse
2) libpil4dfs has high overhead

after that you explain shortlive applications, etc. so you should move the explanation of the exception before 2.
so something like (including some other syntax fixes):

libpil4dfs boosts IO performance by bypassing the fuse kernel when going over dfuse for IO intensive workloads. In some scenarios however, for short running applications or simple processes like linux commands, the IO workload does not justify initializing the DAOS environment in user space with libpil4dfs, since this is relatively expensive. These scenarios can be quantified for example with processes that complete in order of tens or hundreds of milliseconds and run for a large number of times.
To alleviate this problem, DAOS avoid function interception by libpil4dfs for specific executables or commands:
**_List them here please_**
In addition, DAOS provides an environment variable (`D_IL_BYPASS_LIST`) to disable function interception by libpil4dfs for specific applications that are set in that env with the following syntax:
```
$ export D_IL_BYPASS_LIST="app_a:app_b:app_c:app_d"
```

Reply:
Thank you very much for your rewording and example! I will revise them soon.

Reply:
The document is revised. Thank you!